development/src/app/layout.js

```diff
@@ -1,8 +1,13 @@
-"use client";
+'use client';
 
-import "./globals.css";
-import { nohemi } from "@/app/localFont";
-import { ThemeProvider } from "next-themes";
+import './globals.css';
+import { nohemi } from '@/app/localFont';
+import { ThemeProvider } from 'next-themes';
+
+export const metadata = {
```

Comment:
This should be removed, this calls an error for a "use client" having `metadata`